.github/workflows/ci.yml

```diff
@@ -0,0 +1,333 @@
+name: 'ci'
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+  workflow_dispatch:
+env:
+  docker_registry: 'us-docker.pkg.dev'
+  docker_tag: '${{ github.sha }}'
+  docker_repo: 'us-docker.pkg.dev/lumberjack-dev-infra/images'
+  wif_provider: 'projects/638387980668/locations/global/workloadIdentityPools/github-pool/providers/github-provider'
+  wif_service_account: 'gh-access-sa@lumberjack-dev-infra.iam.gserviceaccount.com'
+  server_project_id: 'github-ci-server'
+  client_project_id: 'github-ci-app-0'
+  bigquery_dataset_id: 'audit_logs.audit_abcxyz_data_access'
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+jobs:
+  # Linting jobs - terraform, java, go
+  terraform_lint:
+    uses: 'abcxyz/pkg/.github/workflows/terraform-lint.yml@cc2b918b16df0f627b933ed8dddf0d846fa9108b' # ratchet:abcxyz/pkg/.github/workflows/terraform-lint.yml@main
+    with:
+      directory: 'terraform'
+      terraform_version: '1.2'
+  go_lint_cmd:
```

Comment:
Everything is a single Go module now, so we only need a single `go_lint` and `go_test`: https://github.com/abcxyz/lumberjack/blob/main/.github/workflows/test-unit.yml#L16-L41